src/MicroflowTimer/widget/MicroflowTimer.js

```diff
@@ -73,45 +77,51 @@ mendix.widget.declare("MicroflowTimer.widget.MicroflowTimer", {
 	},
 	
 	execute : function() {
-		if (this.dataobject != null && this.dataobject.getGUID && this.dataobject.getGUID())
+		if (this.dataobject != null && this.dataobject.getGuid && this.dataobject.getGuid())
 		{
 			//microflow set, not already calling a microflow
 			if (this.microflow != '' && this.blocked == false)
 			{
 				this.blocked = true;
-				mx.processor.xasAction({
-					error       : function() {
+				mx.data.action({
+					params: {
+						actionname: this.microflow,
+						applyto: 'selection',
+						caller: this,

```

Comment:
`caller` is deprecated should be `origin: this.mxform`

Reply:
Done